saucal/woocommerce-gateway-amazon-pay
# Tweak/plugin header updates

### All Submissions:






### Changes proposed in this Pull Request:



Adds 3 new headers to every Request to the Amazon Pay API.

### How to test the changes in this Pull Request:

1. Amazon Pay API requests should include the following extra headers.   
`x-amz-pay-platform-version` set to active WooCommerce's version
`x-amz-pay-integrator-version` set to active Amazon Pay for WooCommerce plugin's version
`x-amz-pay-integrator-id` set to the SP's (Solution Provider) ID.

### Other information:




### Changelog entry

> Add - New Amazon Pay Platform related Headers
